source/404.html.md.erb

```diff
@@ -0,0 +1,10 @@
+---
+title: "404"
+last_reviewed_on: 2023-12-19
+review_in: 6 months
+hide_in_navigation: true
+---
+
+# 404 - file not found
```

Comment:
```suggestion
# Page not found
```
To match the [GOV.UK Design System](https://design-system.service.gov.uk/patterns/page-not-found-pages/) guidance :)